observability/readme.md

```diff
@@ -113,3 +113,9 @@ Read more [here](pitfalls.md) to understand what to watch out for while designin
 On-premises Application Insights is useful as a drop-in replacement for Azure Application Insights in scenarios where a solution must be cloud deployable but must also support on-premises disconnected deployment scenarios.
 
 On-premises Application Insights is also useful for testing telemetry integration. Issues related to telemetry can be hard to catch since often these integrations are excluded from unit-test or integration test flows due to it being non-trivial to use a live Azure Application Insights resource for testing, e.g. managing the lifetime of the resource, having to ignore old telemetry for assertions, if a new resource is used it can take a while for the telemetry to show up, etc. The On-premise Application Insights service can be used to make it easier to integrate with an Azure Application Insights compatible API endpoint during local development or continuous integration without having to spin up a resource in Azure. Additionally, the service simplifies integration testing of asynchronous workflows such as web workers since integration tests can now be written to assert against telemetry logged to the service, e.g. assert that no exceptions were logged, assert that some number of events of a specific type were logged within a certain time-frame, etc.
+
+### Azure DevOps Pipelines Reporting with Power BI
+
+The [Azure DevOps Pipelines Report](https://github.com/NarmathaBala/azdo-pipelines-reporting) contains a [Power BI](https://docs.microsoft.com/en-us/power-bi/fundamentals/power-bi-overview) template for monitoring project, pipeline, and pipeline run data from an Azure DevOps (AzDO) organization.
```

Comment:
an extra comma after project?
"for monitoring project pipeline"?

Reply:
The template loads some project information as well so I meant to include the comma. However, since the main focus of the template is on the pipelines and pipeline runs, I'm happy to remove 'project' if you think that would make more sense!